corehq/apps/hqwebapp/static/hqwebapp/js/bootstrap5/common.js

```diff
@@ -3,7 +3,7 @@ hqDefine("hqwebapp/js/bootstrap5/common", [
     'knockout',
     'ko.mapping',
     'underscore',
-    'hqwebapp/js/bootstrap5_loader',
+    'hqwebapp/js/bootstrap5_loader', // note: purposely leaving one reference without es6! prefix to fix loading issues
```

Comment:
What sort of "loading issues" did you encounter that made this necessary? Why does this fix it?

Reply:
@millerdev i don't have a good answer to why it fixes it...just that it does. when this import isn't included without the `es6!` prefix, all the other places where `es6!hqwebapp/js/bootstrap5_loader` is referenced end up returning `undefined` but when this is added back in, then those references return `bootstrap`, however, if those same `es6!` references are changed to `hqwebapp/js/bootstrap5_loader` without the `es6!` prefix, they will ALWAYS return `undefined`